assets/styles/styles.scss

```diff
@@ -3,4 +3,6 @@
 // broken on purpose!
 div {
   border: 2px solid yellow;
+  color: black;
+  background-color: blue;
```

Comment:
Color `blue` should be written in hexadecimal form as `#0000ff`<br>Color literals like `blue` should only be used in variable declarations; they should be referred to via variable everywhere else.